msteams-platform/graph-api/import-messages/update-import-messages.md

```diff
@@ -0,0 +1,373 @@
+
+## Import overview
+
+At a high level, the import process consists of the following:
+
+1. [Create a team with a back-in-time timestamp](#step-one-create-a-team)
+1. [Create a channel with a back-in-time timestamp](#step-two-create-a-channel)  
+1. [Import external back-in-time dated messages](#step-three-import-messages)
+1. [Complete the team and channel migration process](#step-four-complete-migration-mode)
+1. [Add team members](#step-five-add-team-members)
+
+## Necessary requirements
+
+### Analyze and prepare message data
+
+✔ Review the third-party data to decide what will be migrated.  
+✔ Extract the selected data from the third-party chat system.  
+✔ Convert import data into format needed for migration.  
+✔ Map the third-party chat structure to the Teams structure.
+
+### Set up your Office 365 tenant
+
+✔ Ensure that an Office 365 tenant exists for the import data. For more information on setting up an Office 365 tenancy for Teams, *see*, [Prepare your Office 365 tenant](../../concepts/build-and-test/prepare-your-o365-tenant.md).  
+✔ Make sure that team members are in Azure Active Directory (AAD).  For more information *see* [Add a new user](/azure/active-directory/fundamentals/add-users-azure-active-directory) to Azure Active Directory.
+
+## Step One: Create a team
+
+Since existing data is being migrated, maintaining the original message timestamps and preventing messaging activity during the migration process are key to recreating the user's existing message flow in Teams. This is achieved as follows:
+
+1. [Create a new team](/graph/api/team-post?view=graph-rest-beta&tabs=http) with a back-in-time timestamp using the team resource  `createdDateTime`  property.  
+
+1. Place the new team in `migration mode`, a special state that bars users from most activities within the team until the migration process is complete. Include the `teamCreationMode` instance attribute with the `migration` value in the POST request to explicitly identify the new team as being created for migration.  
+
+<!-- markdownlint-disable MD001 -->
+
+#### Permissions
+
+|ScopeName|DisplayName|Description|Type|Admin Consent?|Entities/APIs covered|
+|-|-|-|-|-|-|
+|`Teamwork.Migrate.All`|Manage migration to Microsoft Teams|Creating, managing resources for migration to Microsoft Teams|**Application-only**|**Yes**|`POST /teams`|
+
+#### Request (create a team in migration state)
+
+```http
+POST https://graph.microsoft.com/beta/teams
+
+Content-Type: application/json
+{
+  "@microsoft.graph.teamCreationMode": "migration",
+  "template@odata.bind": "https://graph.microsoft.com/beta/teamsTemplates('standard')",
+  "displayName": "My Sample Team",
+  "description": "My Sample Team’s Description"
+  "createdDateTime": "2020-03-14T11:22:17.067Z"
```

Comment:
CreatedDateTime needs to be unique across messages. This need to be reflected in the document and the examples